pom.xml

```diff
@@ -1,45 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
   <parent>
     <groupId>org.eclipse.jetty.toolchain</groupId>
     <artifactId>jetty-toolchain</artifactId>
-    <version>1.7</version>
+    <version>1.8-parent-SNAPSHOT</version>
   </parent>
-  <modelVersion>4.0.0</modelVersion>
   <groupId>org.eclipse.jetty.toolchain.setuid</groupId>
   <artifactId>jetty-setuid-jna</artifactId>
   <version>2.0.3-SNAPSHOT</version>
-  <name>Jetty :: SetUID JNA</name>
   <packaging>jar</packaging>
+  <name>Jetty :: SetUID JNA</name>
+  <scm child.scm.connection.inherit.append.path="false" child.scm.developerConnection.inherit.append.path="false" child.scm.url.inherit.append.path="false">
+    <connection>scm:git:https://github.com/jetty/${jetty.git.repo}.git</connection>
+    <developerConnection>scm:git:git@github.com:eclipse/${jetty.git.repo}.git</developerConnection>
+    <tag>HEAD</tag>
+    <url>https://github.com/jetty/${jetty.git.repo}</url>
+  </scm>
   <properties>
-    <compiler.source>17</compiler.source>
-    <compiler.target>17</compiler.target>
-    <compiler.release>17</compiler.release>
+    <jdk.version.minimum>17</jdk.version.minimum>
```

Comment:
Where did this property name come from?
Is it one of the bog standard Maven properties?
If not, then this will break some IDE users. (the IDE will not know what JDK version to use)

Reply:
please look at the new parent pom: https://github.com/jetty/jetty.toolchain/blob/7ca85e650cb2b83c80530639685cd5e1255d78b8/pom.xml#L199